vaas/vaas/metrics/tests/test_prometheus.py

```diff
@@ -0,0 +1,79 @@
+from unittest.mock import Mock, call, patch
+from django.test import SimpleTestCase, override_settings
+from prometheus_client import Gauge, Summary
+
+from vaas.metrics.prometheus import PrometheusClient, PrometheusMetrics
+
+
+class TestPrometheusClient(SimpleTestCase):
+    def test_should_get_or_create_metrics(self):
```

Comment:
I would rather test one behavior in one test, for example:
- test_if_create_new_metric_if_not_exists_in_registry
- test_if_retrun_existsing_metric_from_registry

I also recommend the structure [GIVEN-WHEN-THEN ](https://martinfowler.com/bliki/GivenWhenThen.html) for such tests

Reply:
I have split it.